src/features/world/ui/factions/JoinFaction.tsx

```diff
@@ -84,7 +85,7 @@ export const JoinFaction: React.FC<Props> = ({ faction, onClose }) => {
       });
 
       const position = totals.indexOf(faction);
-      const fee = SFL_COST.reverse()[position] ?? 10;
+      const fee = sflCostReverse[position] ?? 10;
```

Comment:
Great spotting on the mutating array!

Would it be possible to perform a shallow clone before the `reverse`, so that the `SFL_COST` array does not get mutated?

```suggestion
      const fee = [...SFL_COST].reverse()[position] ?? 10;
```